pkg/backend/httpstate/backend.go

```diff
@@ -1916,6 +1917,10 @@ func (b *cloudBackend) RunDeployment(ctx context.Context, stackRef backend.Stack
 				colors.Underline+colors.BrightBlue+"%s"+colors.Reset+"\n"), resp.ConsoleURL)
 	}
 
+	if !streamDeploymentLogs {
```

Comment:
Since we inverted the flag, this should now also be inverted.

Reply:
yep